src/box/lua/net_box.c

```diff
@@ -732,6 +733,11 @@ netbox_decode_greeting(lua_State *L)
  * Returns 0 and sets *size to limit or boundary position on success.
  * On error returns -1 and sets diag.
  *
+ * If the connection is closing, then the fiber in which the 'close' function
+ * is called waits on the `on_send_buf_empty` conditional variable until all
+ * data is sent. When all data is send we this variable is signaled and wakeup
```

Comment:
Nit: send ->sent
Nit: seems that 'we' placed in an inappropriate place...

Reply:
Fixed